docs/02-app/01-building-your-application/06-optimizing/06-bundle-analyzer.mdx

```diff
@@ -24,14 +24,12 @@ pnpm add @next/bundle-analyzer
 Then, add the bundle analyzer's settings to your `next.config.js`.
 
 ```js filename="next.config.js"
-const withBundleAnalyzer = require('@next/bundle-analyzer')({
-  enabled: process.env.ANALYZE === 'true',
-})
-
 /** @type {import('next').NextConfig} */
 const nextConfig = {}
 
-module.exports = withBundleAnalyzer(nextConfig)
+const withBundleAnalyzer = require('@next/bundle-analyzer')()
```

Comment:
```suggestion
const withBundleAnalyzer = require('@next/bundle-analyzer')({
  enabled: process.env.ANALYZE === 'true'
})
```
This is still needed https://github.com/vercel/next.js/blob/36e3f4e7abcad8077a687e307ed26805c2eab61c/packages/next-bundle-analyzer/index.js#L6

Reply:
https://github.com/vercel/next.js/blob/36e3f4e7abcad8077a687e307ed26805c2eab61c/packages/next-bundle-analyzer/index.js#L2